### PR TITLE
Microverse: Module1 Capstone

### PR DESCRIPTION
**Pull request by kessie2862**
In this branch, I

- [ ] Included various linter config files
- [ ] Added two HTML pages, **i.e** index.html and about.html
- [ ] Styled the two HTML pages and used media query to make them responsive
- [ ] Added all the project images
- [ ] Included a javascript file containing the functionality for the mobile nav menu and dynamically created the speaker's section.
- [ ] Added the project's live link to the README and a loom video recording describing the project.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+          dedicated to fostering creativity, driving innovation, and promoting
+          effective leadership in today's dynamic and fast-paced business
+          environment."
+        </p>
+        <p class="about-location">
+          Please contact us via email for any further question and
+          clarifications about the ILC Summit 2023
+        </p>
+        <a class="email" href="#">info@ilcsummit2023.org</a>
+      </section>
```
Comment (AbayomiOlaoye) [inline about.html]: - Great job with your implementation here, @kessie2862 :clap: but I believe there are a few touches you can still give this to improve its user interface, as well as improve user experience. Check this comparison out:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|

- Kindly notice the following:
- [ ] [OPTIONAL] The `h1` seems to have a different `font-family`and `font-size` from the requirement
- [ ] Also, it appears the `h1` is not at the center of the columnar section unlike what we have in Cindy's design
- [ ] To also match the design as good as possible, I suggest that you implement that divider between the `title` and the paragraph.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+    <!-- SUB-SECTION STARTS -->
+    <section class="sub-section">
+      <div class="container">
+        <h2>ILC Summit 2023 Logo</h2>
+        <hr />
+        <p>
+          The Logo of the ILC summit 2023 will be decided through the annual ILC
+          logo competition on May 8, 2023.
+        </p>
```
Comment (AbayomiOlaoye) [inline about.html:111]: - [ ] I believe you just merged two sections into one which might not necessarily be the best approach here. How about breaking this up, and then include the currently missing logo that you were introducing in your paragraph just as it is in the design? :hugs: 

_Check this out:_
|Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|

- Looks like title here also has a divider

File: about.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+            <h3 class="summit-header">2021</h3>
+            <p class="summit">ILC Summit 2021 in Cape Coast</p>
+          </div>
+
+          <div class="summit-2">
+            <h3 class="summit-header">2019</h3>
+            <p class="summit">ILC Summit 2019 Accra.</p>
+          </div>
+        </div>
+      </div>
```
Comment (AbayomiOlaoye) [inline about.html:137]: - [ ] As I mentioned above, consider breaking this into another section. That will really make your code more cleanly organized :wink:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-palette"></i>
+              <h3>Design</h3>
+            </div>
+            <p>
+              Get the opportunity to connect with different brands to showcase
+              your creative prowess.
+            </p>
+          </div>
```
Comment (AbayomiOlaoye) [inline index.html:159]: - [ ] Good job, @kessie2862 :clap: However, I suppose these texts can be aligned centrally unlike the `left-alignment` like we have in the template to really make the design more appealing :wink: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|

File: index.html [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+    <!-- MAIN STARTS -->
+
+    <main>
+      <section class="main-page container">
+        <p class="greet">"Lead the world through innovations."</p>
+        <h1>
+          Connecting Minds, <br />
+          Empowering Change <br />
+          2023
+        </h1>
```
Comment (AbayomiOlaoye) [inline index.html]: - Impressive :+1: :clap: :clap: 
- [ ] How about fully implementing appropriate font styles - styling and family, and size for this `h1` element this?

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|

Here's a snippet that could inspire you @kessie2862 :smile: :
```
  h1 {
  font-family: 'cocogoose', sans-serif;
  font-size: 4rem;
  line-height: 5rem;
  padding-top: 1rem;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  background-size: 300%;
  background-image: -webkit-linear-gradient(45deg, #f40001, #ec5242, rgb(255, 64, 0), rgb(255, 123, 0), #a00000, #f65112, #ec5242);
}

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @kessie2862 :wave: , 

Good job with this milestone so far! What a brilliant attempt!!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Great capstone attempt :heavy_check_mark: 
- Video presentation is impressively documentary :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 
- Smoothly responsive webpages :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (kessie2862) [reply about.html]: I don't know if we were given any other font family apart from **LATO** cos that's the font family I used.

Comment (AbayomiOlaoye) [reply about.html]: Oh, wait a minute @kessie2862 I will get you `cocogoose` now. I am sure you will like the feel

Comment (AbayomiOlaoye) [reply about.html]: ...And here is the link to import cocogoose for your use: 
```
@import url('http://fonts.cdnfonts.com/css/cocogoose');

Comment (kessie2862) [reply about.html]: I will appreciate that.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️ ✔️✔️ ✔️

Hi @kessie2862  ✔️ ✔️

Your project is complete! Great job for implementing the changes requested by the previous reviewer🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍✔️ ✔️
- Descriptive Pull Request ✔️ ✔️
- Great design✔️ ✔️
- professional readme✔️ ✔️
- work well presented✔️ ✔️
- No linter errors✔️ ✔️

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**